mathics/builtin/compress.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
-
+"""
+Compress functions
```

Comment:
The pattern for titles that is in use is capitals, e.g. "Compress Functions". See "Form Variables" somewhere else I think in this PR.